comment-popularity.php

```diff
@@ -1,11 +1,10 @@
 <?php
 /*
-Plugin Name: Comment Popularity
-Plugin URI: https://github.com/humanmade/comment-popularity
-Description: Allow visitors to vote on comments.
-Version: 1.4.1
-Author: Human Made Limited
-Author URI: http://humanmade.co.uk
+Plugin Name: Sankaku Comment Popularity
```

Comment:
Sankaku? I think if we're going to take any updates from this PR it'll need to be just the functional parts. Obviously you'd be listed as a contributor but it feels like this PR is deviating quite significantly at this point for it to be something we could merge.